### PR TITLE
WAVE2 1144 Update core courses fields to comp requ fields

### PR DESCRIPTION
**Pull request by robbdavis**
WAVE2 1144 Update core courses fields to comp requ fields

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -304,7 +304,6 @@
  * Completion requirements for thesis based from story #846
  * we will only show completion requirements if author selects "thesis based" course and this field collection is set
  */
-if($content['field_fgs_grad_degree_deliv_type']['#items'][0]['value'] == 'thesis-based') {
   //see if author set any value for completion requirements
   if($content['field_fgs_completion_requiremnts']['#items'][0] != '') {
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: This needs an isset

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install [hunk 2]
```diff
@@ -229,4 +229,77 @@ For more general information, see our <a href="http://www.ucalgary.ca/pubs/calen
...
   }
 
+}
+
+/**
+ * Take core courses fields and put them into the completion requirements fields
+ */
+
+function ucgy_fgs_update_7011() {
+  // Insert the core courses field collection table into the completion requirements field collection table.
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: Stupid question. Looking at this, do we need to do the DB update _before_ we turn conditional fields off?

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php (hunk 1 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Done.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install (hunk 2 above)
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: Yes, each time this code is merged into a new environment, the db update should be run right away.  There are three updates. The first (7011) transfers the course fields content to the requirement fields. The second (7012 deletes the course fields. The third (7013) reverts the feature.

Once all that is run there is still one conditional rule that needs to be deleted in the ui.

Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: Cool. And all that's documented in the release notes, right? :)

Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.install]: Good call. I've updated the release notes to reflect this.

**Pull request merged**